src/api/Feed.js

```diff
@@ -682,15 +704,32 @@ class Feed extends Base {
         this.errorCallback = errorCallback;
         this.updateFeedItem({ isPending: true }, commentId);
 
-        this.commentsAPI.deleteComment({
-            file,
-            commentId,
-            permissions,
-            successCallback: this.deleteFeedItem.bind(this, commentId, successCallback),
-            errorCallback: (e: ElementsXhrError, code: string) => {
-                this.deleteCommentErrorCallback(e, code, commentId);
-            },
-        });
+        const successCallbackFn = this.deleteFeedItem.bind(this, commentId, successCallback);
+        const errorCallbackFn = (e: ElementsXhrError, code: string) => {
+            this.deleteCommentErrorCallback(e, code, commentId);
+        };
+
+        if (shouldUseThreadedComments) {
```

Comment:
We're using `shouldFetchReplies` above. Can we keep the boolean name consistent for future searches?

Reply:
After creating separate methods this is outdated.